origami.json

```diff
@@ -1,5 +1,6 @@
 {
-	"description": "A visual loading indicator",
+	"description": "A visual loading indicator, available in two colours and four sizes",
```

Comment:
> available in two colours and four sizes
I'm not sure this adds much and feels likely to become outdated. Thoughts?

Reply:
I think it's still valuable, there's nothing to explain that that's the case anywhere until you click into the demo